doc/contributing/technical-priorities.md

```diff
@@ -84,34 +100,6 @@ applications on top of the runtime. In addition, it is also important to clearly
 document the use cases, problem determination methods and best
 practices for those tools.
 
-## Permissions/policies/security model
-
-Organizations will only choose technologies that allow them to sufficiently
-manage risk in their production deployments. For Node.js to
-continue its growth in product/enterprise deployments we need to ensure
-that we help them manage that risk. We must have a well-documented
-security model so that consumers understand what threats are/are
-not addressed by the Node.js runtime. We also need to provide
-functions/features which help them limit attack surfaces even if it does
-not result in 100% protection as this will still help organizations
-manage their overall risk level.
-
-## Better multithreaded support
```

Comment:
This one seemed to be mentioned quite a bit in the Community survey so may be worth leaving in.

Reply:
@sheplu I don't think this comment was addressed. Can you take another look.

Reply:
good catch !